src_python/habitat_sim/agent/controls/default_controls.py

```diff
@@ -42,8 +42,9 @@ def _rotate_local(
         if abs(float(rotation.angle())) > 0:
             ref_vector = mn.Vector3()
             ref_vector[axis] = 1
+            abs_rotation_axis = mn.Vector3(np.abs(rotation.axis().normalized()))
 
-            if mn.math.angle(ref_vector, rotation.axis().normalized()) > mn.Rad(1e-3):
```

Comment:
```suggestion
            if mn.math.angle(ref_vector, mn.math.abs(abs_rotation_axis)) > mn.Rad(1e-3):
```

Reply:
Actually never mind, this function isn't exposed on the magum-bindings. Ping @mosra

Reply:
It looks cleaner, but I don't see mn.math.abs in my habitat_sim environment. I might have the wrong version: 
```
>>> dir(mn.math)
['__doc__', '__loader__', '__name__', '__package__', '__spec__', 'acos', 'angle', 'asin', 'atan', 'cos', 'cross', 'dot', 'e', 'inf', 'intersect', 'intersects', 'join', 'lerp', 'lerp_shortest_path', 'nan', 'pi', 'pi_half', 'pi_quarter', 'sin', 'sincos', 'slerp', 'slerp_shortest_path', 'sqrt2', 'sqrt3', 'sqrt_half', 'tan', 'tau']
```

Reply:
You are right, those functions aren't include in the Python binding. Does @erikwijmans `abs(float(mn.math.angle(ref_vector, rotation.axis().normalized())))` diff work? Or is there some subtle missing there? That would be marginally more efficient (since we don't have to reconstruct a mn.Vector3 from a numpy array).

Reply:
Err, wait that won't work nicely either since you need to back into a mn.Rad. I think the original way you did it might be the best until the Magnum bindings are updated.

Reply:
Quite an essential omission, sorry -- I'll add this.

Reply:
@lucasosouza ^

Reply:
As of mosra/magnum-bindings@0120b3f76895efb71f38043638be2f78f31c4ade, the python bindings have [math.abs()](https://doc.magnum.graphics/python/magnum/math/#abs-3776c) alongside other missing scalar/vector functions. Got delayed a bit because the whole world somehow decided to drop Python 3.6 support this week.